src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -92,13 +91,15 @@ public override void OnStartActivity(Activity activity, object payload)
                     // Create a new activity with its parent set from the extracted context.
                     // This makes the new activity as a "sibling" of the activity created by
                     // Asp.Net Core.
-                    Activity newOne = new Activity(ActivityNameByHttpInListener);
+                    Activity newOne = new Activity(ActivityOperationName);
                     newOne.SetParentId(ctx.ActivityContext.TraceId, ctx.ActivityContext.SpanId, ctx.ActivityContext.TraceFlags);
                     newOne.TraceStateString = ctx.ActivityContext.TraceState;
 
                     // Starting the new activity make it the Activity.Current one.
                     newOne.Start();
 
+                    newOne.SetTag("IsCreatedByInstrumentation", bool.TrueString);
```

Comment:
Can we do `SetTag` above the Start() call? This will ensure that this will be the 1st tag. (Because someone could setup a listener to the Start() callback and add a new tag there, which will affect our assumption)

Reply:
This tag is being added to the activity that we create in our callback logic. So even if someone adds a listener to DiagnosticSource start, they will not have access to the activity that we create in our callback method, right?

Reply:
I meant ActivityListener. Anyone can listen to *any* activities, including the ones we create here :)

Reply:
Yeah makes sense. Fixed this.